mm2src/coins_activation/src/tendermint_with_assets_activation.rs

```diff
@@ -38,6 +40,20 @@ impl RegisterTokenInfo<TendermintToken> for TendermintCoin {
     }
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(rename_all = "snake_case")]
+pub enum TendermintPubkeyActivationParams {
+    /// Activate with public key
+    WithPubkey {
+        #[serde(deserialize_with = "deserialize_account_public_key")]
+        pubkey: TendermintPublicKey,
+        is_ledger_connection: bool,
```

Comment:
ooh I see there are more breaking changes, so let's keep them but document the changes. Usually using type and info/data tags for this is prefered so that it's `"type": "wallet_connect"`or `"type": "with_pubkey"` in the request. What do you think?

Reply:
depends on which one will be easier to parse. ( I like both)

```json
"activation_params": {
   "wallet_connect": {
	"session_topic": "bc4c466918fc42ece50c1d01395a218333439464581a7cc6e0da534a9488c3ea"
     }
}
```
OR

```json
"activation_params": {
     "type": "wallet_connect",
      "data":  "bc4c466918fc42ece50c1d01395a218333439464581a7cc6e0da534a9488c3ea" // will be an object for `with_pubkey` by the way,..
}
```

cc. @smk762

Reply:
So update docs to the bottom version?

Reply:
this done @smk762. I updated https://github.com/KomodoPlatform/komodo-defi-framework/blob/wc-integration/mm2src/coins/eth/v2_activation.rs#L167-L167 too

Reply:
> Usually using type and info/data tags

I gave these as examples, but `params` would be more fitting here. And it would be better to leave the WalletConnect Variant as it was with the field `session_topic` `WalletConnect { session_topic: String },`

Reply:
reverted @smk762 please disregard my previous comment above. thanks

Reply:
Sorry if I wasn't clear, I wanted to keep using tag and content but as `#[serde(tag = "type", content = "params")]`, so it should be something like this
```json
"type":"wallet_connect",
"params": {
   "session_topic": "bc4c466918fc42ece50c1d01395a218333439464581a7cc6e0da534a9488c3ea"
}
```

Reply:
re-reverted @smk762 please disregard my previous comment above. thanks